documentation/docs/pmm-upgrade/migrating_from_pmm_2.md

```diff
@@ -17,111 +17,188 @@ Before upgrading to PMM 3, ensure your PMM 2 Server is running the latest versio
 
 ## Step 2: Migrate PMM 2 Server to PMM 3
 
-=== "Automated upgrade (Recommended)"
+=== "Automated Docker upgrade (Recommended)"
     Use this upgrade script for a simplified migration process:
+    { .power-number}
 
-    1. Download and run the following script to start the upgrade. The `-b` flag ensures your data is backed up before the upgrade.
+    1. Download and run the [automated upgrade script](https://raw.githubusercontent.com/percona/pmm/0ad7c05ae253948c779e48ff7976cb5c982af688/get-pmm.sh) to start the upgrade. The `-b` flag creates a backup of your PMM2 instance to ensure that your data is backed up before the upgrade.
```

Comment:
in future it should be replaced by the link used in easy install https://raw.githubusercontent.com/percona/pmm/refs/heads/v3/get-pmm.sh